### PR TITLE
short readme for taskcluster dir

### PR DESCRIPTION
**Pull request by JRMeyer**

### CODE REVIEW

File: taskcluster/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# taskcluster
```
Comment (kdavis-mozilla) [inline taskcluster/README.md]: Just a nit; could you capitalize "t"?

File: taskcluster/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+A task execution framework for Mozilla's CI system. Please consult the [existing taskcluster documentation](https://docs.taskcluster.net/docs).
```
Comment (kdavis-mozilla) [inline taskcluster/README.md]: Could you make it clear that these are files for taskcluster not taskcluster itself.

File: taskcluster/README.md (hunk 1 above)
Comment (JRMeyer) [reply taskcluster/README.md]: commit 54d4c6fa4c0b6d5e21cc42bcfc3d7ba114197dd8

File: taskcluster/README.md (hunk 2 above)
Comment (JRMeyer) [reply taskcluster/README.md]: commit 54d4c6fa4c0b6d5e21cc42bcfc3d7ba114197dd8

**Pull request merged**